exporters/variables-scss/src/generators/__tests__/contentGenerator.test.ts

```diff
@@ -28,6 +29,7 @@ describe('contentGenerator', () => {
         groupNames,
         withCssObject,
         hasParentPrefix,
+        sortByNumValue,
```

Comment:
Well, I think this is another positional parameter, and the number of the variables you must define in the code to be readable suggests, that the object should be used instead of the positional parameters.

```js
generateFileContent({
  tokens: // ...
  mappedTokens: // ...
  // ...
})
```

Reply:
But most of them are optional now and have a default value

Reply:
refactored